korman/exporter/animation.py

```diff
@@ -22,7 +22,7 @@
 
 from . import utils
 
-class AnimationConverter:
+class AnimationConverter(plAGAnim):
```

Comment:
O.o

Reply:
Yeah, I kinda figured that'd catch your eye. Any suggestions on an alternative spot for that...or an alternative callback entirely?

Reply:
I don't think you need this at all?

Reply:
plAGAnim was initially needed for the globalVarName. Otherwise, it couldn't find that property. I may have an alternative now thanks to one of your other suggestions. We'll see if it works.